### ISSUE TITLE
Where do I cp winetricks to on OS X Mountain Lion?

### ISSUE THREAD

**Issue by GoogleCodeExporter** [noahocker/winetricks#374]
```
What steps will reproduce the problem?
1.rm -f winetricks (no feedback)
2.wget http://winetricks.org/winetricks (works fine)
3.sudo cp winetricks /usr/bin *OR* sudo cp winetricks ~/bin (asks for password, 
then no feedback, but this command is the one not working)
4. sudo chmod +x /usr/bin/winetricks (no feedback, but successfully changes 
winetricks to 
an .exe)
5. winetricks command returns wineserver not found! (presumably because it 
wasn't copied into the bin of command exe's

What is the expected output? What do you see instead?
Expect to copy wine tricks into the bin

What version of the product are you using? On what operating system?
WIne 1.7.0 on Mac OS X Mountain Lion with OS X Mavericks and Xcode + command 
line tools installed

Please provide any additional information below.

Basically I'm looking for the directory I need to copy winetricks.exe into, and 
I'm not finding that for Mountain Lion online

I'm trying to reinstall my Direct X drivers to get rid of a *Streaming SIMD 
extensions not supported* error on the Elder Scrolls V: Skyrim Creation kit.

Also when doing the recommended search, I get *syntax error near unexpected 
token fi* and same error for 'done'
```

Original issue reported on code.google.com by `EAschbla...@gmail.com` on 29 Oct 2013 at 5:02

**Comment by GoogleCodeExporter** [comment]
```
How did you install wine?

```

Original comment by `daniel.r...@gmail.com` on 29 Oct 2013 at 5:13

**Comment by GoogleCodeExporter** [comment]
```
uhh I believe I just opened the WIneBottlerCombo 1_4.1.6 and dragged the 
programs into my applications folder as suggested, and it auto updated from 
there. The version in the bottler combo comes with wine tricks, but it only 
seems to work for the version in the combo, which is older, and doesn't run 
things well.
```

Original comment by `EAschbla...@gmail.com` on 29 Oct 2013 at 5:55

**Comment by GoogleCodeExporter** [comment]
```
Couldn't find a bin in there either, even under contents, and the bin in X11 
isn't getting it to work either.
```

Original comment by `EAschbla...@gmail.com` on 29 Oct 2013 at 6:00

**Comment by GoogleCodeExporter** [comment]
```
It's a bit hard for us to support WineBottler.  
See http://winebottler.kronenberg.org/ for support for that package.
It's possible that Mike can answer your question quickly.

Also, isn't 1.4 kind of old?  1.6 is available from 
http://winebottler.kronenberg.org/
```

Original comment by `daniel.r...@gmail.com` on 29 Oct 2013 at 6:11
- Changed state: **Invalid**
